migration_guides/v0.10.2_to_v1.0.0_nearcore.md

```diff
@@ -0,0 +1,375 @@
+-#[derive(BorshSerialize, BorshDeserialize)]
++#[derive(BorshSerialize)]
+ pub struct SignableMessage<'a, T> {
+```
+
+## 6. similar errors
+
+We encounter an error in `near-store` package, similar to above ones.
+It was fixed in a similar way.
+
+### 7. next we encounter an error in `near-network` package:
+
+```rust
+ 1  error: You have to specify `#[borsh(use_discriminant=true)]` or `#[borsh(use_discriminant=false)]` for all enums with explicit discriminant                                                                                                                            ▐
+   --> chain/network/src/types.rs:56:10                                                                                                                                                                                                                                    ▐
+    |                                                                                                                                                                                                                                                                      ▐
+ 56 | pub enum ReasonForBan {                                                                                                                                                                                                                                              ▐
+    |          ^^^^^^^^^^^^                                                                                                                                                                                                                                                ▐
+```
+
+```rust
+#[derive(borsh::BorshSerialize, borsh::BorshDeserialize, Debug, Clone, PartialEq, Eq, Copy)]
+pub enum ReasonForBan {
+    None = 0,
+    BadBlock = 1,
+    BadBlockHeader = 2,
+    HeightFraud = 3,
+    BadHandshake = 4,
+    BadBlockApproval = 5,
+    Abusive = 6,
+    InvalidSignature = 7,
+    InvalidPeerId = 8,
+    InvalidHash = 9,
+    InvalidEdge = 10,
+    InvalidDistanceVector = 11,
+    Blacklisted = 14,
+}
+```
+
+We fix it with `#[borsh(use_discriminant=false)]` as it will result in `12u8` value for `Blacklisted` 
+variant tag, used in serialization, same as before:
+
+```diff
+diff --git a/chain/network/src/types.rs b/chain/network/src/types.rs
+index b2dd97c32..ea2d67f2d 100644
+--- a/chain/network/src/types.rs
++++ b/chain/network/src/types.rs
+@@ -55,2 +55,3 @@ pub struct KnownProducer {
+ #[derive(borsh::BorshSerialize, borsh::BorshDeserialize, Debug, Clone, PartialEq, Eq, Copy)]
++#[borsh(use_discriminant=false)]
+ pub enum ReasonForBan {
+```
+
+### 8. errors, similar to previous ones, in `near-network` and `near-state-viewer` packages
+
+There was a bunch of `borsh::maybestd` imports, which got replaced by their direct from-`std` counterparts.
+
+
+### 9. at this point project compiles, but we have and issue in ci with `cargo-deny --all-features check bans`
```

Comment:
I feel this is out of scope of borsh migration itself, right? I would only mention this in the PR to nearcore to simplify the review.